examples/howto-guides.md

```diff
@@ -0,0 +1,10 @@
+# How-to Guides: Index
+
+* [Running your function as Docker container](/examples/site/howto_create_container/README.md) building and running your
```

Comment:
For all of these bullets, consider some sort of punctuation after the hyperlink. Maybe: `[Title](link) - building and running...`